src/models/board.ts

```diff
@@ -1,15 +1,13 @@
-import Sequelize = require('sequelize');
-
-// const Board = Sequelize.define('board', {
-//   id: {
-//     type: Sequelize.UUID,
-//     defaultValue: Sequelize.UUIDV4,
-//     primaryKey: true,
-//   },
-//   name: Sequelize.STRING(255),
-//   timestamps: true,
-//   createdAt: 'date_created',
-//   updatedAt: 'date_updated',
-// });
-
-// module.exports = Board;
+export const board = (sequelize, Sequelize) => {
```

Comment:
Minor nitpick here: I prefer to handle exports at the bottom of a file rather than inline to make for easier identification of everything that is exported.

There's only one thing exported here, so we could argue that it is fine but I think the consistency is worth it.